### ISSUE TITLE
Add PCRE matching support for string fields

### ISSUE THREAD

**Issue by Totktonada** [tarantool/graphql.0#73]
Extracted from #13.

**Comment by Totktonada** [comment]
Plan:

- [x] PCRE matching for string/string* fields w/o nesting (PR #74).
- [x] PCRE matching for string/string* fields of nested records.
- [ ] PCRE matching for string/string* fields of 1:1-connected records (after #39).

**Comment by Totktonada** [comment]
Sink down the issue to prio2 because of the main part of the plan (the first item) is closed.

**Comment by Totktonada** [comment]
2nd bullet is done in PR #168.

**Comment by Totktonada** [comment]
3rd bullet was extracted to the separate issue: #173.
